ext/ddtrace_profiling_native_extension/native_extension_helpers.rb

```diff
@@ -148,6 +149,10 @@ def self.pkg_config_missing?(command: $PKGCONFIG) # rubocop:disable Style/Global
           "Get in touch with us if you're interested in profiling your app!"
         ].freeze
 
+        UPGRADE_RUBY = [
+          'Upgrade to a modern Ruby to enable profiling for your app.'
```

Comment:
I think "Upgrade to Ruby 2.2 or newer" is more accurate than modern Ruby.

Reply:
Hmmm, I think that may be a bit redundant? We always present the two parts of the error message together, e.g. at installation (which is hidden by default by rubygems):

```
+------------------------------------------------------------------------------+
| Could not compile the Datadog Continuous Profiler because                    |
| the profiler only supports Ruby 2.2 or newer.                                |
|                                                                              |
| The Datadog Continuous Profiler will not be available,                       |
| but all other ddtrace features will work fine!                               |
|                                                                              |
| Upgrade to a modern Ruby to enable profiling for your app.                   |
+------------------------------------------------------------------------------+
```

and at run-time:

```
W, [2022-07-11T09:09:22.971998 #20866]  WARN -- ddtrace: [ddtrace] Profiling was requested but is not supported, profiling disabled: Your ddtrace installation is missing support for the Continuous Profiler because the profiler only supports Ruby 2.2 or newer. Upgrade to a modern Ruby to enable profiling for your app
```

So do you think now that I've changed the first part to already state customers need 2.2+, is it worth repeating it again on the second part?